routes/socket/game/end-game.js

```diff
@@ -17,47 +17,47 @@ const generateGameObject = game => {
 	const casualBool = Boolean(game?.general?.casualGame); // Because Mongo is explicitly typed and integers are not truthy according to it
 	const practiceBool = Boolean(game?.general?.practiceGame);
 	const unlistedBool = Boolean(game?.general?.unlistedGame);
-	const objMap = (obj, f) => new Map(Object.entries(obj).map(([k, v]) => [k, f(k, v)]));
+	const objMap = (obj, f) => new Map(Object?.entries(obj)?.map(([k, v]) => [k, f(k, v)]));
```

Comment:
This is pretty ... interesting 🤔 if we manage to have `Object` be nullish, we might as well give up.